packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,249 @@ def _clean_html(self, html: str) -> str:
-                                        f"Processed {content['type']} '{content['title']}' "
-                                        f"(ID: {content['id']}) from space {self.config.space_key}"
-                                    )
-                            except Exception as e:
-                                logger.error(
-                                    f"Failed to process {content['type']} '{content['title']}' "
-                                    f"(ID: {content['id']}): {e!s}"
+                                logger.debug(
+                                    f"Processed {content['type']} '{content['title']}' "
+                                    f"(ID: {content['id']}) from space {self.config.space_key}"
                                 )
+                        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)